docs/usage/self-hosted-configuration.md

```diff
@@ -292,6 +292,13 @@ Like this:
 
 ## dryRun
 
+You can choose from the following behaviors for the `dryRun` config option:
+
+- `null`: Performs a regular renovate run include creating/updating/deleting branches and PRs, this is the default value
+- `"extract"`: Performs extract only, prints results, exits
+- `"lookup"`: Performs extract and lookup, but exits before running through branches
+- `"full"`: Performs a dry run by logging messages instead of creating/updating/deleting branches and PRs.
```

Comment:
```suggestion
- `"full"`: Performs a dry run by logging messages instead of creating/updating/deleting branches and PRs
```

We don't use punctuation at the end of list items.